package com.google.android.gms.internal.p001firebaseauthapi;

import android.os.RemoteException;
import com.google.android.gms.common.api.Status;
import com.google.firebase.auth.o0;

/* renamed from: com.google.android.gms.internal.firebase-auth-api.zzace  reason: invalid package */
/* compiled from: com.google.firebase:firebase-auth@@22.3.0 */
public interface zzace {
    void zza() throws RemoteException;

    void zza(Status status) throws RemoteException;

    void zza(Status status, o0 o0Var) throws RemoteException;

    void zza(zzaen zzaen) throws RemoteException;

    void zza(zzafj zzafj) throws RemoteException;

    void zza(zzafk zzafk) throws RemoteException;

    void zza(zzafn zzafn) throws RemoteException;

    void zza(zzafn zzafn, zzafc zzafc) throws RemoteException;

    void zza(zzafw zzafw) throws RemoteException;

    void zza(zzagb zzagb) throws RemoteException;

    void zza(zzagj zzagj) throws RemoteException;

    void zza(zzyh zzyh) throws RemoteException;

    void zza(zzyk zzyk) throws RemoteException;

    void zza(o0 o0Var) throws RemoteException;

    void zza(String str) throws RemoteException;

    void zzb() throws RemoteException;

    void zzb(String str) throws RemoteException;

    void zzc() throws RemoteException;

    void zzc(String str) throws RemoteException;
}
